Updating redirect url docs for core 2 (#923)
clerk/clerk-docs: docs/account-portal/custom-redirects.mdx

# Customizing your Account Portal redirects

> When a user clicks on a link or button to navigate to a Clerk sign up or sign in page, Clerk will persist the page the user was on before into a `redirect_url` querystring, and navigate back to that page after the sign up or sign in is completed in order to not interrupt the user's flow through your app. So, for example, if you were on `example.com/foo`, and clicked a "sign in" button on that page, it might take you to `example.com/sign-in?redirect_url=example.com/foo`. After the sign in was complete, you'd be taken back to `example.com/foo`. However, Clerk provides various options for you to customize where your users are redirected to.

Reviewer: ```suggestion
When a user navigates to a Clerk sign up or sign in page via a link or button, Clerk will:
1. Persist the previous page's URL in a `redirect_url` query string
2. Navigate back to that page after the sign-up or sign-in is completed

This avoids interrupting the user's flow through your app. For example, a user selecting a sign in button on `example.com/foo` will navigate to `example.com/sign-in?redirect_url=example.com/foo`, then navigate back to `example.com/foo` upon completing the sign-in process.

The following sections describe the options Clerk provides for you to customize where your users are redirected to upon completing a sign-in or sign-up process.
```

Just breaks the text up and rewords it for easier readability

## Environment variables

You should define the paths you want the users to be redirected to via environment variables. In this example, we redirect users to `/dashboard` after signing in, and to `/onboarding` after signing up, if the user was not already on a page that would be redirected back to.

<CodeBlockTabs type="framework" options={["Next.js", "Remix"]}>
```sh filename=".env"
NEXT_PUBLIC_CLERK_SIGN_IN_FALLBACK_REDIRECT_URL=/dashboard
NEXT_PUBLIC_CLERK_SIGN_UP_FALLBACK_REDIRECT_URL=/onboarding
```

```sh filename=".env"
CLERK_SIGN_IN_FALLBACK_REDIRECT_URL=/dashboard
CLERK_SIGN_UP_FALLBACK_REDIRECT_URL=/onboarding
```
</CodeBlockTabs>

> If you'd like to force the redirect regardless of whether the user was on another page previously, you can use the "force" environment variable values instead as such:

Reviewer: ```suggestion
If you'd like to force the redirect regardless of whether the user was on another page previously, you can use the following `FORCE` environment variable values instead:
```

};
```

## Redirect URL props

> You can explicitly define the redirect paths after sign up or sign in by passing one of the following properties to the respective components:

Reviewer: ```suggestion
You can explicitly define the redirect paths after sign-up or sign-in by passing one of the following properties to the respective components:
```
